static/templates/user_info_popover_content.handlebars

```diff
@@ -12,16 +12,18 @@
             {{/if}}
         </li>
 
-        {{#if is_active }}
-        <li class="user_popover_email">
-            <i class="fa fa-clone hide_copy_icon" data-clipboard-text="{{ user_email }}"></i>
-            {{ user_email }}
-            <div class="tooltip_holder">
-                <span class="email_tooltip">{{ user_email }}</span>
-            </div>
-        </li>
-        {{else}}
-        <li class="user_popover_email half-opacity italic">{{#tr this}}(This user has been deactivated){{/tr}}</li>
+        {{#if show_email}}
+            {{#if is_active }}
+            <li class="user_popover_email">
+                <i class="fa fa-clone hide_copy_icon" data-clipboard-text="{{ user_email }}"></i>
+                {{ user_email }}
+                <div class="tooltip_holder">
+                    <span class="email_tooltip">{{ user_email }}</span>
+                </div>
+            </li>
+            {{else}}
+            <li class="user_popover_email half-opacity italic">{{#tr this}}(This user has been deactivated){{/tr}}</li>
```

Comment:
Do we want to show this deactivated notice with show_email false?  I would imagine so.